src/components/MainView.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Row } from 'reactstrap';
+import ImageManager from './ImageManager';
+
+const MainView = ({
+  heartClass, pressionBottomClass, handleSection, sices, t,
+}) => (
+  <Row className="px-5 pt-5">
```

Comment:
@azizmashkour, what are these classes "px-5 pt-5"? I know the are for padding, but it's better if you wrap the `Row` with `<div className="Main">` and then you have a Main.css file where you define `.Main { ... }`.

Reply:
Ok thanks